hack/tools/go.mod

```diff
@@ -7,10 +7,10 @@ replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.4.2
 require (
 	github.com/drone/envsubst/v2 v2.0.0-20210730161058-179042472c46
 	github.com/golang/mock v1.6.0
-	github.com/golangci/golangci-lint v1.52.2
+	github.com/golangci/golangci-lint v1.53.2
 	github.com/itchyny/gojq v0.12.13
 	github.com/joelanford/go-apidiff v0.5.0
-	github.com/onsi/ginkgo/v2 v2.9.2
+	github.com/onsi/ginkgo/v2 v2.9.4
```

Comment:
why are we updating ginkgo also with this module?

Reply:
my bad, that is getting updated as part of `golangci-lint`, will restrict it to v2.9.2

Reply:
okay, thats fine..